### PR TITLE
add stubs to avaldez1412-phx-az

### LINKED ISSUE #79: add file avaldez1412

### PR DESCRIPTION
**Pull request by avaldez1412**
closes #79

### CODE REVIEW

File: avaldez1412-phx-az.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Phoenix, AZ
```
Comment (githubteacher) [inline avaldez1412-phx-az.md]: How hot is it right now?

**Comment by githubteacher** [comment]
💯 🌵 🦎 💯

File: avaldez1412-phx-az.md (hunk 1 above)
Comment (avaldez1412) [reply avaldez1412-phx-az.md]: 71 Degrees

**Review by githubteacher — APPROVED** [review]
This looks great. I can't believe that it is 70 degrees there. That is exciting!! 👍

**Pull request merged**